### DISCUSSION TITLE
please help change autoload.lua

### DISCUSSION THREAD

**Discussion by ydqsbfsyi** [mpv-player/mpv#14034] [Q&A]
https://github.com/mpv-player/mpv/blob/master/TOOLS/lua/autoload.lua

It's wonderful, just I want the files sorted by date instead of filename, and I want the sort in descending order.

Never learned lua before, so I used AI and I got several versions of code, some got me wrong files count - it's accumulating when I click next button again and again; some failed to show playlist.

Could somebody please give me a hand?

**Comment by Haruki021** [comment]
Maybe you can try to change the sort function's parameters form filename to utils.file_info(path)[＂ctime＂], which function in the autoload.lua
